source/lambda/documentprocessor/lambda_function.py

```diff
@@ -45,11 +62,12 @@ def processRequest(request):
 
     print("Input Object: {}/{}".format(bucketName, objectName))
 
-    ext = FileHelper.getFileExtenstion(objectName.lower())
-    
+    ext = FileHelper.getFileExtension(objectName.lower())
```

Comment:
Should we remove the extension checker altogether and replace it with your mime check?

Reply:
not a bad idea, ive removed it from this lambda. The function itself is used in other places so I've left it in.